src/shared_utilities/Domains.jl

```diff
@@ -926,13 +926,15 @@ function get_additional_domain_fields(subsurface_space)
     z_face = ClimaCore.Fields.coordinate_field(face_space).z
     z_sfc = top_face_to_surface(z_face, surface_space)
     d = depth(subsurface_space)
+    Δz_min = minimum(Δz_top)
     fields = (;
         z = z,
         Δz_top = Δz_top,
         Δz_bottom = Δz_bottom,
         z_sfc = z_sfc,
         depth = d,
         Δz = Δz,
+        Δz_min = Δz_min,
```

Comment:
`Δz_min` here is in `fields`. Is it a Field?

Reply:
no :( but it was a handy place to stick it. maybe I can rename "fields" to "field_data"?

Reply:
To me it looks like it should be related to space/domain

Reply:
Yes, all of these are related to the domain/space. We have domain.space.surface, domain.space.subsurface, domain.fields, etc. The fields are all related to the coordinate field.  So it's just a question of where to stick this additional precomputed quantitity. since it is related to coordinates, I think sticking it in fields is OK, but the name is misleading.

Reply:
I renamed the function to "get_additional_coordinate_field_data" and clarify that `fields` holds both fields and field data.

Reply:
Sounds good!